gatsby/src/components/github/index.js

```diff
@@ -19,7 +18,7 @@ const Github = ({ pageTitle, path }) => {
       <ul className="dropdown-menu" aria-labelledby="dropdownMenu1">
         <li>
           <a
-            href={`https://github.com/pantheon-systems/documentation/edit/master/source/${repoPath}.md`}
+            href={`https://github.com/pantheon-systems/documentation/edit/master/source/_docs/${path}.md`}
```

Comment:
@jmolivas This is the path for content in Sculpin. Shouldn't it be `/source/content/${path}.md` in anticipation of Gatsby?

Reply:
@alexfornuto The path of this link is pointing to the repository to open the file on edit mode

![image](https://user-images.githubusercontent.com/366275/61245948-8c78a200-a702-11e9-8119-4750f7c6c19b.png)

Reply:
I believe my point remains valid - once `gatsby` is `master`, that path changes.